.github/workflows/slack_notifications.yml

```diff
@@ -35,7 +36,10 @@ jobs:
             rake
             redis
             resque
-            unicorn"
+            roda
+            ruby-openai
```

Comment:
Thanks for getting this update started, Hannah! I have two questions:
1. Should we alphabetize this list? (it's so close!)
2. Do we also want to include stripe?